frontend/src/metabase/query_builder/components/MoveQuestionModal/MoveQuestionModal.tsx

```diff
@@ -204,9 +204,13 @@ export const MoveQuestionModal = ({
         onClose={onClose}
         title={
           <Title fz="1.25rem" lh={1.5}>
-            Moving this question to another dashboard will remove it from{" "}
-            <Icon name="dashboard" style={{ marginBottom: -2 }} size={20} />{" "}
-            <Dashboards.Name id={question.dashboardId()} />
+            {c("{0} is the dashboard name")
```

Comment:
```suggestion
            {c("{0} is the name of a dashboard")
```

Reply:
I'm just thinking that if we refer to "the dashboard", the translator may wonder, "What dashboard?"